scripts/performance_comparison.sh

```
@@ -0,0 +1,68 @@
+        echo "🎯 Testing: $filename"
+        echo "-------------------"
+        
+        # Test interpreter
+        echo -n "📝 Interpreter: "
+        INTERPRETER_TIME=$(( time cargo run --release --bin trbi "$file" > /dev/null 2>&1 ) 2>&1 | grep real | awk '{print $2}')
+        echo "$INTERPRETER_TIME"
+        
+        # Test compiler (if compilation succeeds)
+        echo -n "⚡ Compiler: "
+        if cargo run --release --bin trbc -- --compile "$file" -o "/tmp/tribute_test_$filename" > /dev/null 2>&1; then
+            COMPILER_TIME=$(( time cargo run --release --bin trbc -- --compile "$file" -o "/tmp/tribute_test_$filename" > /dev/null ) 2>&1 | grep real | awk '{print $2}')
```

Shellcheck (0.10.0)

[error] Shells disambiguate $(( differently or not at all. For $(command substitution), add space after $( . For $((arithmetics)), fix parsing errors.

(SC1102)